### DISCUSSION TITLE
Help to customize a theme for @carbon/react

### DISCUSSION THREAD

**Discussion by sildeman** [carbon-design-system/carbon#11574] [Help]
How do I create a theme with my brand colors, to apply in a @carbon/react application?

I tried to follow the directions on this page: https://carbondesignsystem.com/guidelines/themes/overview/#customizing-a-theme, but I didn't get the expected result.  However, I confess that I'm not very good with styles and I don't know anything about sass.

If anyone has an example to help me, I'd be grateful.

Thanks!

**Comment by tay1orjones** [comment]
Hey @sildeman sorry for the confusion on this one. We're still working on getting docs consistent across the website and repository, #11293 

We cover this a bit in the [package sass docs](https://github.com/carbon-design-system/carbon/blob/864e94a66e375cf6ae16bf54dcd42317c0970519/packages/themes/docs/sass.md), but you should be able to extend and override the theme with your own tokens or values:

```scss
@use '@carbon/themes/scss/themes';
@use '@carbon/themes' with (
  $fallback: themes.$g100,
  $theme: (
    background: #000000,
    or-a-new-token: #000000,
  )
);
```

**Reply by sildeman** [reply]
Yes, I'm using this strategy to apply my color setup. 

In fact I would like to know if it's possible (in a simple way) to create a complete theme (example: $my-theme) and use it together with the Theme component, as if it were a native carbon theme (`<Theme theme='my-theme'>`), or use it as the main theme and apply the native carbon themes to parts of the interface (`<Theme theme='g100'>`).

Thanks
